src/Runner/Hook.php

```diff
@@ -114,16 +133,13 @@ public function run(): void
      */
     protected function handleAction(Config\Action $action): void
     {
-        $this->io->write(['', 'Action: <comment>' . $action->getAction() . '</comment>'], true, IO::VERBOSE);
```

Comment:
Why did you remove it? Now it is duplicated in line 137 and 142.

Reply:
The difference is the verbosity level. I find it useful when the action name gets printed out even in normal verbosity level (so I can understand which CLI output comes from which CLI command), but skipped actions can be quiet on normal verbosity level.

Nevertheless, this more belongs to another pull request improving the console output.